### DISCUSSION TITLE
New baby

### DISCUSSION THREAD

**Discussion by onyebuchinzube247** [community/community#56892] [Programming Help]
### Select Topic Area

Product Feedback

### Body

I need an expert on JavaScript, java, css please who is ready to help me become a programmer, I will be glad to archive my dream on GitHub, thanks

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @onyebuchinzube247 ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by lancao2** [comment]
how can I help you?

**Reply by onyebuchinzube247** [reply]
I need to learn JavaScript, I need a guider and a teacher

**Comment by onyebuchinzube247** [comment]
I want to be a JavaScript professional, I will be glad if you help me
archive the goal, thank you and have a nice day

**Reply by emilburganov** [reply]
Try watching tutorial videos and doing similar projects

**Comment by onyebuchinzube247** [comment]
Thanks, I'll check it out.
